modules/sriov-operator-metrics.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="sriov-operator-metrics_{context}"]
+= Enabling the SR-IOV Network Operator metrics
+
+You can enable the SR-IOV Network Operator metrics to monitor and analyze the Operator's performance. 
```

Comment:
This doesn't look accurate to me. The feature allows the user to monitor workload pod networking activity, not the operator's one.

Reply:
Updated as per the suggestion.